src/conf/startup.properties

```diff
@@ -136,6 +136,12 @@ prism.configstore.listeners=org.apache.falcon.entity.v0.EntityGraph,\
 
 *.falcon.cleanup.service.frequency=days(1)
 
+# Default number of threads to be used to restore entities.
+*.falcon.default.threads.load.restore.entities=100
+
+# Default timeout in minutes to load entities
+*.falcon.default.timeout.minutes=30
```

Comment:
In large organizations/cluster with many feed in the range of 3-4K takes time if the persistent store is hdfs for the prism-store. 
Anyways this is only default value for timeout, if the entities are very few, it will just finish off and restore them